dbus_next/message.py

```diff
@@ -135,7 +135,7 @@ def new_error(msg: 'Message', error_name: str, error_text: str) -> 'Message':
                        body=[error_text])
 
     @staticmethod
-    def new_method_return(msg: 'Message', signature: str = '', body: List[Any] = []) -> 'Message':
+    def new_method_return(msg: 'Message', signature: str = '', body: List[Any] = [], fds: List[Any] = []) -> 'Message':
```

Comment:
Be more specific than `List[Any]`. I think you only accept a union of `socket` and `int`.

Reply:
This could be something else other than a socket so I changed it to List[Union[int, IO]]